ra/ra.go

```diff
@@ -90,20 +91,49 @@ func NewRegistrationAuthorityImpl(
 		log:   logger,
 		authorizationLifetime:        DefaultAuthorizationLifetime,
 		pendingAuthorizationLifetime: DefaultPendingAuthorizationLifetime,
-		rlPolicies:                   policies,
-		tiMu:                         new(sync.RWMutex),
-		maxContactsPerReg:            maxContactsPerReg,
-		keyPolicy:                    keyPolicy,
-		maxNames:                     maxNames,
-		forceCNFromSAN:               forceCNFromSAN,
-		regByIPStats:                 scope.NewScope("RA", "RateLimit", "RegistrationsByIP"),
-		pendAuthByRegIDStats:         scope.NewScope("RA", "RateLimit", "PendingAuthorizationsByRegID"),
-		certsForDomainStats:          scope.NewScope("RA", "RateLimit", "CertificatesForDomain"),
-		totalCertsStats:              scope.NewScope("RA", "RateLimit", "TotalCertificates"),
+		rlMu:                 new(sync.RWMutex),
```

Comment:
Sure - that makes sense. Its error prone (as I discovered myself) having the mutex carried at the side of the object being synchronized. I'll rework.